raiden/transfer/mediated_transfer/events.py

```diff
@@ -66,18 +70,32 @@ class SendBalanceProof(Event):
     Note:
         This event has a dual role, it serves as a synchronization and as
         balance-proof for the netting channel smart contract.
+
+        Nodes need to keep synchronized the latest known merkle root, this is
```

Comment:
Please rephrase to:
` Nodes need to keep synchronized the latest known merkle root, this is ` -> `Nodes need to keep the last known merkle root synchronized`.

Also stop the sentence there and start a new sentence.